src/public/modules/core/css/avatar-dropdown.css

```diff
@@ -134,6 +125,9 @@
   border-radius: 50%;
   outline: none;
   margin-right: 16px;
+  display: inline-table;
```

Comment:
why `inline-table` rather than `inline`?

Reply:
Played with the CSS, inline-table is the only one that correctly allows avatars to take up their full 40px width LOL